client/asset/btc/btc.go

```diff
@@ -976,6 +976,11 @@ func (btc *ExchangeWalletSPV) RemovePeer(addr string) error {
 func (btc *ExchangeWalletFullNode) FeeRate() uint64 {
 	// NOTE: With baseWallet having an optional external fee rate source, we may
 	// consider making baseWallet a FeeRater by allowing a nil local func.
+	if btc.walletInfo.Name == "Firo" {
+		// estimatesmartfee unstable for firo, estimatefee not much better ;(
```

Comment:
Is this a widely-known or documented issue?  For most assets, btc included, estimatesmartfee doesn't want to work unless the chain is well utilized and the node has warmed up for over an hour.  This means that these RPCs typically don't work on simnet, often not on testnet either.

Reply:
Related, looks like we can use "https://explorer.firo.org/insight-api-zcoin/utils/estimatefee" for an `ExternalFeeEstimator`

Reply:
Updates as discussed: in https://github.com/decred/dcrdex/pull/2270/commits/cb14f9a01722cc7a0e62d4def5971b7bb8767fc9